client/src/unifyfs.c

```diff
@@ -1899,6 +1904,7 @@ int unifyfs_fid_open(
      *   check that local and global info are consistent
      * if O_APPEND, set pos to file size
      */
+    int need_truncate = 0;
 
```

Comment:
Separating this line of code from the preceding comment, which is independent.
```suggestion
     */
     
     /* flag indicating whether file should be truncated */
    int need_truncate = 0;

```